src/Content/Entity/Conversation/Channel.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2023, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Content\Entity\Conversation;
+
+/**
+ * @property-read string $code        Channel code
+ * @property-read string $label       Channel label
+ * @property-read string $description Channel description
+ * @property-read string $accessKey   Access key
+ */
+final class Channel extends \Friendica\BaseEntity
```

Comment:
The presence of the access key and the name of the array that contains them make me think this would be better named `ChannelTab`.

Furthermore, I think it's better if the path is `Content/Conversation/Entity/ChannelTab.php`.

Reply:
The plan is, that eventually there is a table in the background that holds the user defined channels. So I would like to keep that name.

Reply:
The general naming pattern I've used is the following: `<Category of the feature>/<Feature>/<Class type>/<Entity/Factory/Repository name>`

If you start with the class type, you'll potentially end up with many entities that don't have much with each other.